### DISCUSSION TITLE
Firebird Auto Increment does not exists, hide the option

### DISCUSSION THREAD

**Discussion by adaliszk** [dbeaver/dbeaver#111] [Ideas]
In Firebird (least in 2.5 and below) the Auto Increment does not exists for collumn property. The Unique ID generated by Generator in before insert trigger.

**Comment by serge-rider** [comment]
Yes, but what is this issue about? Auto-increment is always off so it doesn't look like a bug.
For now Firebird uses Generic driver which shows the same properties for all databases. Auto-increment is a standard read-only column property.

**Comment by adaliszk** [comment]
Sorry, I mean It should not showing the collumn because the whole feature is does not exists in Firebird.

**Comment by serge-rider** [comment]
Agreed, but while Firebird driver is based on Generic driver it will show all standard things. I don't want to over-complicate Generic driver, it is already too sophisticated.
Just ignore this auto-increment thing :)
